test/processors/image_processor_test.rb

```diff
@@ -7,7 +7,7 @@ class ImageProcessorTest < ActiveSupport::TestCase
     ImageProcessor.supported_tasks.first.must_equal 'resize'
   end
 
-  describe "resize_image" do
+  describe 'resize_image' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.